UEBPrep/app/models/user.rb

```diff
@@ -62,6 +62,7 @@ def self.find_for_twitter_oauth(auth, signed_in_resource=nil)
 
         user = User.create(provider:auth.provider,
                            uid:auth.uid,
+                           username:auth.uid+"@twitter.com",
                            email:auth.uid+"@twitter.com",
```

Comment:
This data should be transparent to the user in any case if they are using the social media login. Question is, can they log in at all without something in this field? It's either bad data or no data and no data = no login, but maybe I am wrong.